packages/native/test/native-styled.test.js

```diff
@@ -136,4 +136,15 @@ describe('Emotion native styled', () => {
 
     expect(tree).toMatchSnapshot()
   })
+
+  it('Log error message if units are not specified when using shorthand properties', () => {
+    const Text = styled.Text`
+      margin: 20px;
+      padding: 50;
+    `
+
+    const tree = renderer.create(<Text>Hello World</Text>).toJSON()
+
+    expect(tree).toMatchSnapshot()
+  })
```

Comment:
Could you make it so this checks the error message works? something like this.
https://github.com/emotion-js/emotion/blob/76828f1f8394af31f046085cc07e259b92f56341/packages/emotion/test/warnings.test.js